steps/test/unit/tReorderMSData.cc

```diff
@@ -21,32 +22,61 @@ using dp3::steps::Step;
 
 BOOST_AUTO_TEST_SUITE(reorder)
 
-bool compareBinaries(const std::string& filename1,
-                     const std::string& filename2) {
-  std::ifstream file1(
-      filename1,
+// Function to compare two binary files. If the files for comparison are Meta
+// file, the flag isMetaFile is set to true.
+std::pair<std::istreambuf_iterator<char>, std::istreambuf_iterator<char>>
+compareBinaryFiles(const std::string& referenceFile,
```

Comment:
For new code (like everything in this file) please adhere to naming conventions from google c++ style guide https://google.github.io/styleguide/cppguide.html#Variable_Names

e.g. input_file not inputFile please pay attention also to the rule of Minimize the use of abbreviations while reading over the style guide.

If modifying existing functions that use old conventions then you can in those cases (at your discretion, if it makes sense) stick to the existing style within those functions for consistency instead.